user_guide_src/source/changelogs/v4.3.6.rst

```diff
@@ -24,6 +24,8 @@ Deprecations
 Bugs Fixed
 **********
 
+- **Connection:** Fixed a bug where semicolon character (`;`) in one of the Postgre connection parameters would break the DSN string.
```

Comment:
```suggestion
- **Database:** Fixed a bug where semicolon character (``;``) in one of the Postgre connection parameters would break the DSN string.
```